rpc/documentation/api.md

```diff
@@ -334,6 +334,10 @@ should begin at.
 - `int32 first_new_block_height`: The height of the first new block added to the
   main chain.  Only non-zero when `fetched_headers_count` is not zero.
 
+- `bytes main_chain_tip_block_hash`: The hash of the main chain tip block.
```

Comment:
be a bit more explicit here in the description, it is the current main chain tip block after any new headers have been fetched and applied to the main chain.